### ISSUE TITLE
Are there plans to make this transparently work with `snowfallorg/lib`?

### ISSUE THREAD

**Issue by mainrs** [snowfallorg/nix-software-center#69]
To me it seems like `lib` kind of exists for the purpose of this tool to work in a more structured way. Installing packages can now be done by creating a module inside `nixos`/`homes` and activating said module inside the final host configuration.

**Comment by yonacwy** [comment]
Yeah i saw this too im interested to see integrations in the next release, but i don't see any info on a next release.
